crates/networking/p2p/Cargo.toml

```diff
@@ -40,4 +40,4 @@ rand = "0.8.5"
 hex-literal = "0.4.1"
 
 [lib]
-path = "./net.rs"
+path = "./lib.rs"
```

Comment:
Shouldn't it be `p2p.rs` to match the other crates?

Reply:
Im moving all to `lib.rs` as discussed the daily.

Reply:
But maybe I should do it all together in the same PR, switching back to p2p.rs